fcrepo-http-api/src/test/java/org/fcrepo/integration/http/api/FedoraLdpIT.java

```diff
@@ -2634,7 +2631,8 @@ public void testBinaryLastModified() throws Exception {
         final long lastmod1;
         try (final CloseableHttpResponse response = execute(putDSMethod(objid, "binary1", "some test content"))) {
             assertEquals(CREATED.getStatusCode(), getStatus(response));
-            lastmod1 = headerFormat.parse(response.getFirstHeader("Last-Modified").getValue()).getTime();
+            lastmod1 = Instant.from(
+                headerFormat.parse(response.getFirstHeader("Last-Modified").getValue())).toEpochMilli();
```

Comment:
You should be able to define `lastmod1` as an `Instant`, and then compare the values below with `lastmod1.isBefore(lastmod2)` -- rather than converting all the `Instant`s to `long`s